spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -102,11 +102,11 @@ with other specifications of MicroProfile.
 
 ===== Required Base metrics
 
-Each vendor of a MicroProfile-compliant server can implement setting up the metrics in the _base_ scope in a
-vendor-specific way.
+Required base metrics describe a set of metrics that all MicroProfile-compliant servers have to provide.
+Each vendor can though implement the set-up of the metrics in the _base_ scope in a vendor-specific way.
```

Comment:
"though" sounds a bit awkward.
Maybe "Each vendor though, can implement.." or preferably "Each vendor can implement"

Reply:
/me is not a native English speaker :)